packages/flame/lib/src/camera/camera_component.dart

```diff
@@ -271,4 +271,10 @@ class CameraComponent extends Component {
       boundedBehavior.bounds = bounds;
     }
   }
+
+  /// Returns true if the given [component] is currently visible through this
+  /// camera.
+  bool inView(PositionComponent component) {
+    return visibleWorldRect.overlaps(component.toAbsoluteRect());
```

Comment:
We can probably make this more efficient in the long run, but it might not be possible now.

Reply:
Would it be useful to return the overlapping `Rect` instead of a `bool`? Maybe we need to make a decision according to how much it is being "seen"?

Reply:
That could probably be another helper method.